ui/src/LandingPageContent.vue

```diff
@@ -0,0 +1,294 @@
+
+<template>
+    <!-- subtract 62px for the header and 10 vh for the footer -->
+    <el-main class="main">
+        <el-row class="hero-banner-container">
+            <el-col :xs="0" :md="1" />
+            <el-col :xs="24" :md="10" class="landing-page-col">
+                <el-row>
+                    <h1 class="hero-banner-title">
+                        Convert neuroimaging data and associated metadata to the BIDS
+                        standard
+                    </h1>
+                </el-row>
+                <el-row>
+                    <p class="hero-banner-description">
+                        ezBIDS requires neither coding proficiency nor knowledge of BIDS in
+                        order to get started. It is the first BIDS tool to offer guided
+                        standardization, support for task events conversion, and
+                        interoperability with OpenNeuro and brainlife.io.
```

Comment:
Let's provide links to OpenNeuro and brainlife, e.g.:

`interoperability with <a href="https://openneuro.org/" target="_blank" >OpenNeuro</a> and <a href="https://brainlife.io/about/" target="_blank" >brainlife.io</a>.`

Reply:
done!

Reply:
@nicoalee how about re-directing to this brainlife link: https://brainlife.io/about/

Reply:
For those unfamiliar with the brainlife platform

Reply:
Disregard, I'll merge the PR and add that in myself.